environments/openstack/playbook-bootstrap-basic.yml

```diff
@@ -194,6 +194,14 @@
         url: http://cloud-images.ubuntu.com/minimal/releases/focal/release/ubuntu-20.04-minimal-cloudimg-amd64.img
         dest: /tmp/ubuntu.img
 
+    - name: Get timestamp from the system
+      shell: "date +%Y-%m-%d%H-%M-%S"
```

Comment:
There seems to be a space missing and the time is misformated.
'+%Y-%m-%d %H:%M:%S'. However, my recommendation would be to leave out the time and just use '+%Y-%m-%d'.